src/EditorFeatures/Core.Wpf/Suggestions/AsyncSuggestedActionsSource.cs

```diff
@@ -118,7 +119,7 @@ private partial class AsyncSuggestedActionsSource : SuggestedActionsSource, IAsy
                                 state, requestedActionCategories, document,
                                 range, selection,
                                 addOperationScope: _ => null,
-                                includeSuppressionFixes: priority.Value == CodeActionRequestPriority.Normal,
+                                includeSuppressionFixes: priority.Value == CodeActionRequestPriority.Low,
```

Comment:
tbh, i would also jsut love a dedicate call to get suppression fixes.  it would make more sense for me if we legit just had the lightbulb pull that as the very last thing it did.  I'm not a huge fan of making the same call sseveral times, passing a bool which all but one call use to not do work, and one call does to only do work :)

Reply:
or just pass in the priority.  have the lower layer decide if that means suppression or not.  it would be breate if hte action-soruce had no concept of suppressions at all.

Reply:
I'll do the latter, thanks.